filebeat/docs/modules-overview.asciidoc

```diff
@@ -0,0 +1,182 @@
+[[filebeat-modules-overview]]
+== Modules overview
+
+beta[]
+
+Filebeat modules simplify the collection, parsing, and visualization of common
+log formats.
+
+A typical module (say, for the Nginx logs) is composed of one ore
+more filesets (in the case of Nginx, `access` and `error`). A fileset contains
+the following:
+
+* Filebeat prospector configurations, which contain the default paths were to
+  look or the log files. These default paths depend on the operating system.
+  The Filebeat configuration is also responsible with stitching together
+  multiline events when needed.
+
+* Elasticsearch {elasticsearch}/ingest.html[Ingest Node] pipeline definition,
+  which is used to parse the log lines.
+
+* Fields definitions, which are used to configure Elasticsearch with the
+  correct types for each field. They also contain short descriptions for each
+  of the fields.
+
+* Sample Kibana dashboards, which can be used to visualize the log files.
+
+Filebeat automatically adjusts these configurations based on your environment
+and loads them to the respective Elastic stack components.
+
+NOTE: At the moment, Filebeat modules require using the Elasticsearch
+{elasticsearch}/ingest.html[Ingest Node]. In the future, Filebeat Modules will
+be able to also configure Logstash as a more powerful alternative to Ingest
+Node.
+
+=== Tutorial
+
+This tutorial assumes you have Elasticsearch and Kibana installed and
+accessible from Filebeat (see the <<filebeat-getting-started,getting started>>
+section). It also assumes that the Ingest Node GeoIP and User Agent plugins are
+installed, which you can do with the following two commands executed in the
+Elasticsearch home path:
+
+[source,shell]
+----------------------------------------------------------------------
+$ sudo bin/elasticsearch-plugin install ingest-geoip
+$ sudo bin/elasticsearch-plugin install ingest-user-agent
+----------------------------------------------------------------------
```

Comment:
should we mention that ES must be restarted afterwards? (or is this not the case?)